lib/mainloop.c

```diff
@@ -494,6 +494,12 @@ main_loop_reload_config(MainLoop *self)
   return;
 }
 
+void
```

Comment:
I think this commit is not needed, as the next commit just redefines this function.

Reply:
I will squash all intermediate commits and will push again soon.